soh/soh/Enhancements/game-interactor/GameInteractor.h

```diff
@@ -302,6 +302,8 @@ typedef enum {
     // Opt: *EnFr
     // Vanilla condition: this->reward == GI_NONE
     VB_FROGS_GO_TO_IDLE,
+    // Vanilla condition: var >= gSaveContext.health) && (gSaveContext.health > 0
+    VB_PREVENT_ALARM_AT_FULL_HEALTH,
```

Comment:
Probably a more accurate name for this VB:
```suggestion
    VB_HEALTH_METER_BE_CRITICAL,
```

Reply:
I'm all for it. I'm terribad with names!